code/modules/surgery/rig_removal.dm

```diff
@@ -0,0 +1,58 @@
+//Procedures in this file: Unsealing a Rig.
```

Comment:
The code was lacking any way to remove a sealed rigsuit. If the wearer died, the rigsuit would have been lost for the station, while the normal spacesuit can just be stripped with a longer delay. I couldn't think of any other way to remove a rigsuit from someone than the surgery, so i used it.